packages/block-library/src/index.js

```diff
@@ -104,66 +104,82 @@ const registerBlock = ( block ) => {
 	registerBlockType( name, settings );
 };
 
+/**
+ * Function to get all the core blocks in an array.
+ *
+ * @example
+ * ```js
+ * import { __experimentalGetCoreBlocks } from '@wordpress/block-library';
+ *
+ * const coreBlocks = __experimentalGetCoreBlocks();
+ * ```
+ */
+export const __experimentalGetCoreBlocks = () => [
```

Comment:
I'm not convinced it needs to be experimental, but it does no harm (other than later maintenance).